src/mbgl/renderer/tile_mask.hpp

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <mbgl/tile/tile_id.hpp>
+
+#include <set>
+
+namespace mbgl {
+
+// A TileMask is a set of TileIDs that describe what part of a tile should be rendered. I.e. it
+// denotes the part of a tile that is covered by other/better tiles. If the entire tile should be
```

Comment:
nit: these first two sentences seem to contradict each other — first that these parts of a tile "should be rendered" and then that this mask denotes parts that should be _not_ be rendered